values.yaml

```diff
@@ -348,7 +348,8 @@ puppetdb:
   #    cpu: 1000m
   ## Additional puppetdb container environment variables
   extraEnv: {}
-
+  metrics:
+    enabled: false
```

Comment:
The indentation doesn't seem right.
And if you could put a comment above those two lines with a little bit info what it enables. 

Also - in `README.md` could you please put info for those new parameters (there is such a section towards the end of the file).

Reply:
Sure, I will.